int16 sampleTimeType
float64 value
